changefeedccl: remove rotted TODOs

danhhz opened the pull request:
Also resolve an easy one.

Release note: None

danhhz committed:
changefeedccl: remove rotted TODOs

Also resolve an easy one.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/36295)
<!-- Reviewable:end -->

danhhz committed:
changefeedccl: don't sleep before first table history update

This order might shave a tiny bit of time off test runtime and no reason
to do it the order it was before.

Release note: None

tbg reviewed (approved):
> Reviewed 6 of 6 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/36295)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @nvanbenschoten)



<!-- Sent from Reviewable.io -->

danhhz commented:
Thanks for the review!

bors r=tbg
